ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/redundancy.j2

```diff
@@ -1,7 +1,7 @@
-{% if redundancy is defined and redundancy is not none %}
+{% if redundancy is arista.avd.defined %}
```

Comment:
Add comment at the beginning of the file to identify functionality (consistency between files)

Reply:
addressed with commit: eeca525